pkg/build/types/image_configuration_test.go

```diff
@@ -208,7 +208,7 @@ func TestMergeInto(t *testing.T) {
 			StopSignal: "bar",
 			WorkDir:    "bar",
 			Accounts: types.ImageAccounts{
-				RunAs: "bar",
```

Comment:
Why do `WorkDir`, `Cmd`, etc., all get `"bar"` when this should get `"foo"`?

This code is weird.